src/runtime.rs

```diff
@@ -1,15 +1,28 @@
-use anyhow::{anyhow, Result};
 use docker_api::Docker;
+
+// Imports not used by windows environments
+#[cfg(not(target_os = "windows"))]
 use podman_api::Podman;
+#[cfg(not(target_os = "windows"))]
+use anyhow::Result;
+#[cfg(not(target_os = "windows"))]
+use anyhow::anyhow;
```

Comment:
Do we need to gate these imports? I imagine we can use anyhow regardless of the target OS?

Reply:
Yep, so as the comment says, these are imports that windows environments don't use. We can totally import these regardless of the environment but if you're working on `dcp` in a windows environment, or build it in a windows environment, then you'll get an error here for unused imports. Its just a warning though, so nothing breaking.

Reply:
Ohh interesting. I would prefer no warnings, so let's keep this as-is.